core/src/main/scala/io/delta/tables/DeltaTable.scala

```diff
@@ -618,7 +618,9 @@ object DeltaTable {
   }
 
   /**
-   * Create a DeltaTable for the data at the given `path`.
+   * Instantiate a DeltaTable representing the data at the given `path`, If the given path is
+   * invalid (i.e. either no table exists or an existing table is not a delta table), it throws
+   * a `notADeltaTableException` Exception.
```

Comment:
Instantiate a [[DeltaTable]] object representing the data at the given `path`, If the given path is
invalid (i.e. either no table exists or an existing table is not a Delta table), it throws a 'not a Delta table`
error.

Reply:
updated